src/js/components/Meta.js

```diff
@@ -192,7 +207,22 @@ class Meta extends React.PureComponent {
     return this.setHukamnamaAudioPlayerVisibility(e);
   }
 
-
+  async componentDidMount() {
+    if (this.props.type === 'shabad' ) {
+      const healthy = await checkAPI()
+      if (healthy) {
+        const audioUrl = await getAudio(this.props.info);
```

Comment:
@MySmilingTurban better to name this function as `getShabadAudioUrl` or `getPanktiAudioUrl`